packages/mongodb-memory-server-core/src/util/getos/index.ts

```diff
@@ -131,6 +131,14 @@ export function isValidOs(os: LinuxOS | undefined): os is LinuxOS {
   return !isNullOrUndefined(os) && os.dist !== UNKNOWN;
 }
 
+export function isDebian10OrHigher(os: AnyOS): boolean {
+  if (!isLinuxOS(os) || os.dist !== 'debian') {
+    return false;
+  }
+
+  return ['unstable', 'testing', ''].includes(os.release) || parseFloat(os.release) >= 10;
+}
+
```

Comment:
i dont think this is the proper place for this, `getos` is intended to get all information for a given system, but not modify it or distro specific functions, i think the better place would be in `MongoBinaryDownloadUrl` or `utils`

Reply:
Removed.